apache/trafficcontrol
# Fix DS SSL keys not updating when updating CDN

## What does this PR (Pull Request) do?


- [x] This PR fixes #1847


## Which Traffic Control components are affected by this PR?


- Traffic Control Client (Go) 
- Traffic Ops

## What is the best way to verify this PR?
Given two CDNs `x1` and `x2`:
-  Create a DS `y` on CDN `x1`
-  Generate SSL keys for `y`
-  Update `y` to be on CDN `x2`
-  Verify SSL keys are now on `x2` and **not** `x1`

Run the api tests and verify that they pass. **Note:** the ssl keys test are marked as a system test as they need riak to work.




## If this is a bug fix, what versions of Traffic Control are affected?

master (55b3b03)

## The following criteria are ALL met by this PR
